_data/places.yml

```diff
@@ -305,7 +325,7 @@
   url: https://orders.dickeys.com/quote/location
   instagram: https://www.instagram.com/dickeys.bk/
   phone: (347) 889-5400
-  hours: 10AM - 10PM
+  hours: 10AM – 10PM
```

Comment:
What is this change? Replacing a dash with an emdash?

Reply:
Yeah, regular hyphens with en-dashes. Can revert to hyphens again if we prefer, it was mostly an effort to have consistent naming, casing and spacing.

Reply:
Totally in favor of consistent naming, spacing, etc., but unsure what the benefit of en-dashes is? Seems like slower more friction to maintain but I don’t feel strongly and am open to being convinced otherwise!

Reply:
Gonna change it back for easier contribution. Grammatically it’s more correct (ranges should be expressed with en-dashes) but stylistically it’s a tad trickier to write.

Reply:
TIL! Didn’t realize that hyphens weren’t grammatically correct – great to know. Thank you!